### PR TITLE
Fix application of pagination options on ListCopilotSeats endpoint

### PR DESCRIPTION
**Pull request by ryanskidmore**
The pagination options were previously being applied as the Request body, instead of applied to the URL as in other `List` endpoints. This PR ensures that these options are correctly applied to the URL and introduces additional parameters in the test case to ensure this.

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @ryanskidmore !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Ah, whups - my bad, codecov reminded me that we need some more tests.
I'll point you to an example.  One moment...

File: github/copilot_test.go
Comment (gmlewis) [inline github/copilot_test.go:480]: In this test, we will also want to add:

```go
	opts := &ListOptions{Page: 1, PerPage: 100}
	got, _, err := client.Copilot.ListCopilotSeats(ctx, "o", opts)
...
	const methodName = "ListCopilotSeats"
	testBadOptions(t, methodName, func() (err error) {
		_, _, err = client.Copilot.ListCopilotSeats(ctx, "\n", opts)
		return err
	})

	testNewRequestAndDoFailure(t, methodName, client, func() (*Response, error) {
		got, resp, err := client.Copilot.ListCopilotSeats(ctx, "o", opts)
		if got != nil {
			t.Errorf("Copilot.ListCopilotSeats returned %+v, want nil", got)
		}
		return resp, err
	})
```

Comment (ryanskidmore) [reply github/copilot_test.go]: Added these in - ty for the heads up. I don't think I can trigger the codecov workflow so I'm not sure if this has increased the coverage by enough!

**Comment by ryanskidmore** [comment]
> Thank you, @ryanskidmore ! LGTM.
> 
> Awaiting second LGTM+Approval from any other contributor to this repo before merging.

Hey @gmlewis just wanted to follow up on this. I noticed codecov is still failing, do I need to add any more tests to this?

**Comment by gmlewis** [comment]
> Hey @gmlewis just wanted to follow up on this. I noticed codecov is still failing, do I need to add any more tests to this?

No, codecov has been pretty unreliable lately. You can ignore it in this case.

**Review by be0x74a — APPROVED** [review]
LGMT! 🚀

**Comment by gmlewis** [comment]
Thank you, @be0x74a !
Merging.

**Pull request merged**
